test/PushNotificationTests/APITests.cpp

```diff
@@ -77,150 +99,96 @@ namespace Test::PushNotifications
         TEST_METHOD_SETUP(MethodInit)
         {
             VERIFY_IS_TRUE(TP::IsPackageRegistered_ProjectReunionFramework());
-
-            m_failed = CreateTestEvent(c_testFailureEventName);
-
             return true;
         }
 
         TEST_METHOD_CLEANUP(MethodUninit)
         {
             VERIFY_IS_TRUE(TP::IsPackageRegistered_ProjectReunionFramework());
 
+            if (m_process)
+            {
+                VERIFY_IS_TRUE(CloseHandle(m_process));
+            }
             return true;
         }
 
-        TEST_METHOD(ChannelRequestUsingNullRemoteId)
+        void RunTest(const PCWSTR& testName)
         {
-            wil::unique_event event = CreateTestEvent(c_testProtocolScheme_Packaged);
+            DWORD pid;
+            VERIFY_SUCCEEDED(m_testAppLauncher.get()->ActivateApplication(L"PushNotificationsTestAppPackage_8wekyb3d8bbwe!App", testName, AO_NONE, &pid));
+            VERIFY_IS_NOT_NULL(pid);
+
+            m_process = OpenProcess(SYNCHRONIZE | PROCESS_QUERY_LIMITED_INFORMATION, FALSE, pid);
+            VERIFY_IS_NOT_NULL(m_process);
 
-            // This is associated protocol for the MSIX installed app for launch.
-            // Use the ://path to define the component you want to test.
-            Uri launchUri{ c_testProtocolScheme_Packaged + L"://ChannelRequestUsingNullRemoteId" };
-            auto launchResult = Launcher::LaunchUriAsync(launchUri).get();
-            VERIFY_IS_TRUE(launchResult);
+            VERIFY_ARE_EQUAL(WaitForSingleObject(m_process, 3000), WAIT_OBJECT_0);
 
-            WaitForEvent(event, m_failed);
+            DWORD exitCode;
+            VERIFY_IS_TRUE(GetExitCodeProcess(m_process, &exitCode));
```

Comment:
Here we want to verify a BOOL. So let's use this TAEF macro:

```suggestion
            VERIFY_WIN32_BOOL_SUCCEEDED(GetExitCodeProcess(m_process, &exitCode));
```